src/main/java/org/skriptlang/skriptworldguard/elements/expressions/ExprRegionFlag.java

```diff
@@ -0,0 +1,131 @@
+package org.skriptlang.skriptworldguard.elements.expressions;
+
+
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+import com.sk89q.worldguard.WorldGuard;
+import com.sk89q.worldguard.protection.flags.*;
+
+import com.sk89q.worldguard.protection.regions.ProtectedRegion;
+
+
+import org.bukkit.event.Event;
+import org.skriptlang.skriptworldguard.SkriptWorldGuard;
+import org.skriptlang.skriptworldguard.worldguard.WorldGuardRegion;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ExprRegionFlag extends PropertyExpression<WorldGuardRegion, String> {
+
+    static {
+
+        register(ExprRegionFlag.class, String.class, "[worldguard] flag %string%", "worldguardregions");
+
+    }
+
+    private Expression<String> flag;
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public boolean init(Expression<?>[] expression, int matchedPattern, Kleenean kleenean, ParseResult parseResult) {
+        flag = (Expression<String>) expression[0];
+        setExpr((Expression<? extends WorldGuardRegion>) expression[1]);
+        return true;
+
+    }
+
+    @Override
+    protected String[] get(Event e, WorldGuardRegion[] region) {
+        Flag<?> fl = null;
+
+
+        WorldGuard wg = WorldGuard.getInstance();
+        fl = Flags.fuzzyMatchFlag(wg.getFlagRegistry(), flag.getSingle(e));
+
+        List<String> finalv = new ArrayList<>();
+
+        for (WorldGuardRegion wrg : region){
+            ProtectedRegion rg = wrg.getRegion();
+            if (rg != null){
+                finalv.add(rg.getFlag(fl).toString());
+            }
+        }
+
+
+
+        return (String[]) finalv.toArray();
+    }
+
+    @Override
+    public String toString(Event e, boolean debug) {
+
+        return "flag \"" + flag.getSingle(e) + "\" of region" ;
+    }
+
+    @Override
+    public Class<? extends String> getReturnType() {
+        return String.class;
+    }
+
+
+
+    @Override
+    public Class<?>[] acceptChange(final Changer.ChangeMode mode){
+        if (mode == Changer.ChangeMode.SET || mode == Changer.ChangeMode.DELETE) { return CollectionUtils.array(String.class);}

```

Comment:
Please read our [code conventions](https://github.com/SkriptLang/Skript/blob/master/code-conventions.md)
It's preferred to use switch statement here instead of if-else blocks
```suggestion
        if (mode == Changer.ChangeMode.SET || mode == Changer.ChangeMode.DELETE) {
                return CollectionUtils.array(String.class);
        }
```